src/misc/ci-cycles.md

```diff
@@ -0,0 +1,67 @@
+# Continuous Integration Cycles
+
+When working on Yorick, it's possible to create "CI cycles".
```

Comment:
This sentence feels like it ended too soon? Maybe something like "Since Yorick is split over different repositories, it is possible to get into a seeming deadlock between repositories, where merging a PR in one seems to depend on a PR in the other -- and vice versa. Fortunately, there is a solution."

Reply:
The following section explains this. I would have killed the sentence, but it looks weird to have no text between a major section and a subsection.

If you don't think it looks weird, we can drop this sentence.

Reply:
I actually tried something else. Let me know what you think.